src/content/docs/launch/index.mdx

```diff
@@ -29,38 +31,29 @@ For complete documentation on launching your storefront, visit the [Launch secti
 
 During the development phase of your project, you'll encounter tasks around migrating implementations that were done on a development or staging environment to production. It is essential to keep track of these changes and make them reproducible on the production environment as part of the launch activities.
 
 :::caution
 Avoid scheduling a launch shortly before a weekend. The Adobe engineering team might have limited availability to fix critical issues if they occur.
 :::
 
-Start your checklist with the [Go-Live Checklist](https://www.aem.live/docs/go-live-checklist). For Commerce sites, you should prioritize SEO tasks. Additionally, consider the following action items.
-  
-* Switch to the Catalog Service production endpoint (https://catalog-service.adobe.io/graphql)
-  * Ensure a production environment is configured in the Service Connector (will result in a new `environmentId`).
-  * Sync the production catalog to the new production environment.
-  * Create a new Commerce production API key-pair and use the public key as the `x-api-key` value.
-  * Double check category IDs and make sure all category pages have a reference to the correct category.
-  * Update `environmentId` and `x-api-key` values in the `.helix/config.xlsx` file.
-  * Let the Catalog Service team know about the new production environment and launch date.
-
-* Use the production GraphQL endpoint `yourproject.com/graphql` (needs to be changed in the CDN).
-  * Make sure to also use this for all the Sidekick extensions and scripts (e.g. sitemap generation, image importer).
-
-* When using Adobe Commerce Fastly, request a new CDN purge token for Commerce production environment.
-  * Update in Sharepoint in the `.helix/config.xlsx` file (`authToken` and `serviceId`).
-
-* Verify/update links for password reset e-mail to match the Edge Delivery Service implementation in the Adobe Commerce Admin.
-* Verify the configuration of integrations and payment providers (reCAPTCHA, Adyen, Bazaarvoice).
-  * Provide and configure production keys.
-  * Verify that the new domains/subdomains are allowlisted and potential backend webhooks are working.
-
-* Add a `robots.txt` file.
-  * Block the `/drafts` path on the CDN to prevent any drafts from being publicly accessible on the production URL.
-  
-* Ensure that the site's Lighthouse score is green; targeting `100` on every page (taking into account previous considerations mentioned on this document).
+Start your checklist with the [Go-Live Checklist](https://www.aem.live/docs/go-live-checklist). For Commerce sites, you should prioritize SEO tasks. Additionally, make sure that you have completed the following items.
+
+<Checklist tasks={[
```

Comment:
Additionally I would like to add:

* We can combine the catalog service tasks (production environmentId & key + production endpoint and config update) into a single action item as it's all related and should be done in a single step.
* Additional item: For multi store setups, ensure that you add a store specific cache buster (e.g. query parameter or proxy via CDN) to Catalog Service and LiveSearch requests. (link to https://wiki.corp.adobe.com/pages/viewpage.action?pageId=3072519697#:~:text=Important%20for%20multi%20store%20setups%20or%20stores%20supporting%20multiple%20locales%3A) 
* Regarding the robots.txt:
    > Add a robots.txt file to your project which allows your site to be indexed by search engines. Ensure that your sitemaps are referenced and that you added rules to block indexing of any content you do not wish to be indexed (e.g. /drafts folder).
* Additional item: Adding one or multiple [redirect](https://www.aem.live/docs/redirects) files to ensure that URLs that were changed as part of the migration still work (e.g. when you remove `.html`).
* Additional item: Have a sitemap generated of your site and catalog. To speed up the indexing process we recommend adding it to Google Search Console.
* Additional item: Ensure that your product detail pages have metadata and structured data (e.g. LD-JSON) configured. (link to https://experienceleague.adobe.com/developer/commerce/storefront/seo/metadata/)
* Additional item: Validate your storefront events implementation and ensure data is displayed in your LiveSearch and Product Recommendation dashboards in Commerce admin. (@sirugh and @shiftedreality can provide documentation links).
* Additional item: Validate your CDN configuration and ensure that caching and cache invalidation works as expected (link to Fastly docs).

Reply:
For storefront events: https://github.com/adobe/commerce-events/tree/main/examples/events/snowplow-debugger